### ISSUE TITLE
App closing out at the 2 hour mark

### ISSUE THREAD

**Issue by fieroloki** [chtaylo2/Roku-GooglePhotos#447]
We noticed this started this past Friday. At almost exactly 2 hours, the app closes out back to the Roku home screen on all 4 TV's we have.
 Release v3.7.2 - Build 5.620.20210301

I do see the Roku devices did an update Friday, but the power/data saving settings are intact and set to off.

Edit: This appears to be a Roku issue itself.  I have even tried the new Roku photo streams and it does the same thing.

**Comment by FavaTeaMarketing** [comment]
We are having a similar issue and have the same release/build.

**Comment by louisecasey** [comment]
I have seen the same thing.  I need to keep a loop repeating all day and it no longer does that.

**Comment by Diaa-G** [comment]
I have the same problem here, the app closes after 2 hours on roku tv

**Comment by ajbplayluv** [comment]
We are seeing the same issue on the 2 hour mark. Any word on a fix?

**Comment by Cris000077** [comment]
We are having the same issue on the 2 hour mark. I tried the #375 bug fix and it didn't work for me is there something new to help out?

**Comment by fieroloki** [comment]
I found a workaround.  Set your screensaver timeout as low as it will go without being disabled. When the 2-hour limit hits the screens will still go back to the home screen, but it will automatically start again.

**Comment by DCJeff** [comment]
also having the screen timeout at the 2-hour mark even though it's on only 1 photo.

**Comment by creed1010** [comment]
We use this app on several TVs and they all are timing out even faster now.  Some after an hour and a half.  Do we think there will be a fix for this?  I'm trying to decide whether I need to come up with another solution for us.  If you think you can fix this soon, let me know and I'll make a good donation once it is up and running again.

**Comment by RevRumbley** [comment]
Same here!! For about the past year I was able to have uninterrupted slideshow playback in our church foyer on a Roku TV using the PhotoView app with the simplicity of drag and drop pics into Google Photos from my office. About 5 weeks ago it started timing out after about 2 hours and kicking out to the Roku menu. It is unusable to me in this state and I would love to see it fixed and would donate to see it fixed! It seems like the support is no longer there to expect updates or a fix for this. Is that true??? Thanks!  OR - does anyone here know of another suitable alternaive app?

**Comment by chtaylo2** [comment]
Acknowledging the issue.  I’ve not had a chance to look into this yet. Roku changes things all the time and keeping up with it is tough. I’ll look I to it soon

**Comment by RevRumbley** [comment]
Thanks! Appreciate the response and any attention to this. LOVE THE APP! Can't wait for the fix! :)

**Comment by DCJeff** [comment]
After getting the same thing with PLEX, I read that Roku last month disabled independent apps from overriding the screensaver time. They had to pull that setting in their last update.

**Comment by creed1010** [comment]
I understand.  Thank you for looking into this.  Your work on this is much
appreciated!

**Comment by lucarsen** [comment]
Hi,

Is there anything new on this issue? 

Thank you.

**Comment by RevRumbley** [comment]
Yes - any updates???  I haven't seen any difference in the behavior of the app yet - I try about once per week. It still times out around the 2 hour mark. Unfortunately we are moving away from using this app out of necessity until something changes. I can't wait for it to work again. Please help. Thanks!

**Comment by fieroloki** [comment]
> Yes - any updates??? I haven't seen any difference in the behavior of the app yet - I try about once per week. It still times out around the 2 hour mark. Unfortunately we are moving away from using this app out of necessity until something changes. I can't wait for it to work again. Please help. Thanks!

By chance did you see my other reply?   "I found a workaround. Set your screensaver timeout as low as it will go without being disabled. When the 2-hour limit hits the screens will still go back to the home screen, but it will automatically start again."

**Comment by RevRumbley** [comment]
I did not see that, I'm trying it right now, thank you. But we were able to use this basically as remotely manageable digital signage previously without any interruptions. We'll really need that kind of functionality again for it to work the way we need in our context. Thanks!

**Comment by creed1010** [comment]
We tried that workaround already a few weeks ago on several TVs and it
didn't fix it for us.

Thanks,

Chris Reed
Technology Director
ASI Gymnastics
‎214.280.9777

**Comment by tnemidur** [comment]
Any word on if this will be fixed or has anyone found another work around?

**Comment by tdawg97** [comment]
following as well. having the same issues.

**Comment by alexort11** [comment]
Same problem here with about 10 screens. Hoping for a fix.

**Comment by robcookies** [comment]
Love the app and it's been hugely helpful, but I've been having this same issue for the last 3-4 months.

**Comment by louisecasey** [comment]
"At almost exactly 2 hours, the app closes out back to the Roku home screen".   Has this issue ever been resolved?

**Comment by creed1010** [comment]
It still does the same thing for us so no it has not been fixed.

**Comment by fieroloki** [comment]
The workaround I mentioned is still working for us.  
"Set your screensaver timeout as low as it will go without being disabled. When the 2-hour limit hits the screens will still go back to the home screen, but it will automatically start again."

**Comment by creed1010** [comment]
Yep, already tried that and didn't fix it for us.

**Comment by louisecasey** [comment]
This works, kind of.  Thanks.  Did you find a way to hold pictures more than 30 seconds (after the initial 2 hours).  

    On Friday, December 1, 2023 at 07:37:02 AM PST, fieroloki ***@***.***> wrote:  
 
 


The workaround I mentioned is still working for us.
"Set your screensaver timeout as low as it will go without being disabled. When the 2-hour limit hits the screens will still go back to the home screen, but it will automatically start again."

**Comment by FavaTeaMarketing** [comment]
We have gone to the Roku photo app and have had no issues.

From: louisecasey ***@***.***>
Date: Monday, December 4, 2023 at 6:09 PM
To: chtaylo2/Roku-GooglePhotos ***@***.***>
Cc: Fava Tea Marketing ***@***.***>, Comment ***@***.***>
Subject: Re: [chtaylo2/Roku-GooglePhotos] App closing out at the 2 hour mark (Issue #447)
This works, kind of.  Thanks.  Did you find a way to hold pictures more than 30 seconds (after the initial 2 hours).

On Friday, December 1, 2023 at 07:37:02 AM PST, fieroloki ***@***.***> wrote:




The workaround I mentioned is still working for us.
"Set your screensaver timeout as low as it will go without being disabled. When the 2-hour limit hits the screens will still go back to the home screen, but it will automatically start again."

**Comment by RevRumbley** [comment]
I hate to say it, but we have moved on from this app that I loved due to the 2 hour issue and the lack of any resolution. Once it stopped working, we never could get the workarounds to do what we needed. It is time to fix this.. or we should all figure out something else that does what we need.

**Comment by creed1010** [comment]
Yes, we are going to drop the app as well.  We are moving to a full digital
signage solution.  It costs more and doesn't integrate with Google Photos
but it integrates with Google Docs so it will at least work without timing
out.

Thanks,

Chris Reed
Technology Director
ASI Gymnastics
‎214.280.9777

**Comment by shawarmahousewaterdown** [comment]
Hi,
I have the same issue , can you please provide me solution or workaround or any other software apps for Roku.
Thanks
